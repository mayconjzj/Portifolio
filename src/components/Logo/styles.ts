import tw from 'tailwind-styled-components';

export const Logo = tw.span`
  font-black text-[23px] text-blue-900
`;

export const ContentLogo = tw.div`
  flex justify-center flex-col
`;

export const Description = tw.span`
  text-blue-900
`;
