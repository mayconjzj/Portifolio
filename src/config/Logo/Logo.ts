import { LogoProps } from '@/models/Logo';

export const LogoConfig: LogoProps = {
  ContentLogo: 'Maycon Douglas',
  Description: 'Desenvolvedor Front-End'
};
