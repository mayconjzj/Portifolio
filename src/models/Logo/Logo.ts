export type LogoProps = {
  ContentLogo: string;
  Description: string;
};
