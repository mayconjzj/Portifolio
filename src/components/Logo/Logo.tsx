import * as S from './styles';

export const Logo = () => {
  return (
    <S.ContentLogo>
      <S.Logo>MAYCON JZJ</S.Logo>
      <S.Description>Desenvolvedor Front-End</S.Description>
    </S.ContentLogo>
  );
};
