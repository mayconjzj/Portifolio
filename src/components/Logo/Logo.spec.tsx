import { render } from '@testing-library/react';

import { Logo } from './Logo';

describe('Logo', () => {
  it('should render the logo', () => {
    render(<Logo />);
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toBeTruthy();

    expect(asFragment()).toMatchSnapshot();
  });
});
