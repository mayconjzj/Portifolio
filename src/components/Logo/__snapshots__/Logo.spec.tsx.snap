// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render the logo 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center flex-col"
  >
    <span
      class="font-black text-[23px] text-blue-900"
    >
      MAYCON JZJ
    </span>
    <span
      class="text-blue-900"
    >
      Desenvolvedor Front-End
    </span>
  </div>
</DocumentFragment>
`;
